src/experimental-bots/swift/placerExample.ts

```diff
@@ -360,7 +369,7 @@ export class SwiftPlacer {
 					}
 
 					if (resp.simError) {
-						logger.info(`${logPrefix}: ${resp.simTxLogs}`);
+						logger.info(`${logPrefix}: ${resp.simError}, ${resp.simTxLogs}`);
```

Comment:
```suggestion
						logger.info(`${logPrefix}: ${JSON.stringify(resp.simError)}, ${resp.simTxLogs}`);
```

otherwise will get [object object]